vispy/visuals/isocurve.py

```diff
@@ -96,8 +106,8 @@ def set_data(self, data):
         if _HAS_MPL:
             try:
                 if not (self._X.T.shape == data.shape):
-                    raise
-            except:
+                    raise Exception
+            except Exception:
```

Comment:
There is actually a difference, `except:` will even catch `SystemExit` whereas `except Exception:` will not. It might also work for KeyboardInterrupt or whatever that one is called.